config/environments/production.rb

```diff
@@ -76,7 +76,7 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
-  config.action_mailer.default_url_options = { host: "admin.wifi.service.gov.uk" }
+  config.action_mailer.default_url_options = { host: ENV["FQDN"] }
```

Comment:
With reference to the PR on the Terraform PR, if we could use more readable names it would help document the code.
```suggestion
  config.action_mailer.default_url_options = { host: ENV["FULL_DOMAIN_NAME"] }
```

Reply:
fixed it to FULLY_QUALIFIED_DOMAIN_NAME. hope that is clearer. (it should include the hostname as well)